CMakeLists.txt

```diff
@@ -85,6 +87,19 @@ if(NOT CC_OPTIMIZE)
   endif()
 endif()
 
+# TOOD: Need more test before it works
```

Comment:
If this already works, can this commented code be removed, or do more tests still need to be added?

Reply:
keep it as a TODO reminder to have a new PR of continuous work on this code snippet.